ckanext/reclineview/theme/public/vendor/recline/recline.js

```diff
@@ -2619,7 +2622,10 @@ my.MultiView = Backbone.View.extend({
         </div> \
       </div> \
       <div class="recline-results-info"> \
-        <span class="doc-count">{{recordCount}}</span> records\
+      {{#recordCountWasEstimated}} \
+        <span class="doc-count-approx">about</span> \
+      {{/recordCountWasEstimated}} \
+      <span class="doc-count">{{recordCount}}</span> records \
```

Comment:
"about" and "records" need to be translated here.

Would be nice to truncate the estimate too, e.g. `about 315k records`

Reply:
What about just using [tilde (~)](https://en.wikipedia.org/wiki/Tilde)?

Reply:
Recline is not translated anyway - I guess there are JS reasons for it. So I'd rather leave that to a separate effort [
<img width="954" alt="screen shot 2018-10-12 at 13 56 23" src="https://user-images.githubusercontent.com/307612/46870532-65f9a380-ce1e-11e8-8b05-4e003cb43a3c.png">
](https://demo.ckan.org/it/dataset/newcastle-city-council-payments-over-500/resource/0c2bd47a-6ac5-412e-a337-1b45a952e07e)